plugins/cypressRemarkPlugins/.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
What does this get us?

Reply:
ahh, probably shouldn't of been checked in. This was before I moved the plugins to be their own package and I set this up to debug the ts code.

Reply:
i removed it